advocacy_docs/supported-open-source/patroni/debian11_quick_start.mdx

```diff
@@ -167,7 +171,7 @@ EOF
 
 `$MY_IP` and `$MY_NAME` are specific to the local host. Otherwise, the `patroni.yml` configuration is the same on all Patroni nodes.
 
-Patroni expects to find the `postgres` binary in the `bin_dir` location, while the EDB Postgres Advanced Server binary is called `edb-postgres`. Create a symbolic link to let Patroni find that binary:
+Patroni expects to find the `postgres` binary in the `bin_dir` location, while the EDB Postgres Advanced Server binary is called `edb-postgres`. Before Patroni [version `3.0.3`](https://github.com/zalando/patroni/blob/master/docs/releases.rst#version-303), remove the `bin_name` setting from the configuration and create a symbolic link to let Patroni find that binary:
```

Comment:
```suggestion
Patroni expects to find the `postgres` binary in the `bin_dir` location, while the EDB Postgres Advanced Server binary is called `edb-postgres`. As `postgresql.bin_name` setting was introduced in [version `3.0.3`](https://github.com/zalando/patroni/blob/master/docs/releases.rst#version-303), if you are running Patroni older than that, then remove the `bin_name` setting from the configuration and create a symbolic link to let Patroni find that binary:
```

I wonder if this extra verbosity makes it a bit more friendly?

Reply:
Hm :thinking:, I don't really like to repeat the setting name again. Maybe:

_As the `postgresql.bin_name` setting was introduced in [version `3.0.3`](https://github.com/zalando/patroni/blob/master/docs/releases.rst#version-303), remove it from the configuration if you are running an older Patroni version and create a symbolic link..._